### PR TITLE
Show error in "Profile Name" field if `sso-session` already exists

### PR DESCRIPTION
**Pull request by vchikoti1998**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description
Display a profileName field error in the IDC setUpAuthentication Dialog. If a user enters the same profile name as one that already exists.

Screenshot 2023-09-14 at 10 06 49 AM

## Checklist
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: resources/resources/software/aws/toolkits/resources/MessagesBundle.properties [hunk 1]
```diff
@@ -909,6 +909,7 @@ gettingstarted.setup.iam.profile=Profile Name:
 gettingstarted.setup.iam.profile.comment=Used by AWS Toolkit to list added credentials
 gettingstarted.setup.iam.secret_key=Secret Access Key:
 gettingstarted.setup.idc.profile.comment=User-specified name used to label credentials locally
+gettingstarted.setup.idc.profile.error_message=This profileName is already exists
```
Comment (manodnyab) [inline resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:912]:
```suggestion
gettingstarted.setup.idc.profile.error_message=Profile name already exists
```

Comment (vchikoti1998) [reply resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:912]: Sure, will update the sentence

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/SetupAuthenticationDialog.kt [hunk 2]
```diff
@@ -274,6 +275,7 @@ class SetupAuthenticationDialog(
             textField()
                 .comment(message("gettingstarted.setup.idc.profile.comment"))
                 .errorOnApply(message("gettingstarted.setup.error.not_empty")) { it.text.isBlank() }
+                .errorOnApply(message("gettingstarted.setup.idc.profile.error_message")) { ProfileFile.defaultProfileFile().profiles().containsKey(it.text) }
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/SetupAuthenticationDialog.kt:278]: this isn't quite right, could you add tests?

Comment (vchikoti1998) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/SetupAuthenticationDialog.kt:278]: sure, definitely

Comment (vchikoti1998) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/SetupAuthenticationDialog.kt:286]: As discussed offline with Richard, suggested that should only be looking at SSO sessions name. It's not about whether there's a profile with the same name; it's about finding an SSO session with a matching name.

**Pull request closed without merging**